### ISSUE TITLE
4090 support

### ISSUE THREAD

**Issue by Elliot-Construct** [DualCoder/vgpu_unlock#120]
Will this ever get 4090 support?

**Comment by thisIsLoading** [comment]
my 6x RTX 4090 AI workstation is desperately waiting for it

**Comment by alpozcelik05** [comment]
Sadly not yet supporting. GTX 1XXX and RTX 2XXX series supports

**Comment by delta-whiplash** [comment]
> my 6x RTX 4090 AI workstation is desperately waiting for it

do you have deals by any chance ?

**Comment by codgician** [comment]
I think support is unlikely... Starting from Ampere (RTX30xx) vGPU implementation has become SR-IOV based instead of time slicing. SR-IOV support should be enabled in GPU firmware for that to function properly (which unfortunately is not enabled for consumer grade GPUs). GPU firmware is signed nowadays making flashing modded firmware very hard.

**Comment by olljanat** [comment]
Has anyone investigated options which Nvidia supports nowadays for those newer cards? Based on https://nvidia.custhelp.com/app/answers/detail/a_id/5173/~/geforce-gpu-passthrough-for-windows-virtual-machine-(beta) at least scenario where host system is Linux and guest Windows should be supported but is it also possible to have Windows+Hyper-V as host and Linux as guest?

Of course there is that single VM limitation so it is not solution for everyone.

**Comment by Rikitik** [comment]
Monday, August 21st 2023
[NVIDIA BIOS Signature Lock Broken, vBIOS Modding and Crossflash Enabled by Groundbreaking New Tools](https://www.techpowerup.com/312631/nvidia-bios-signature-lock-broken-vbios-modding-and-crossflash-enabled-by-groundbreaking-new-tools). There is still hope i guess

**Comment by Alfatango12** [comment]
> Monday, August 21st 2023 [NVIDIA BIOS Signature Lock Broken, vBIOS Modding and Crossflash Enabled by Groundbreaking New Tools](https://www.techpowerup.com/312631/nvidia-bios-signature-lock-broken-vbios-modding-and-crossflash-enabled-by-groundbreaking-new-tools). There is still hope i guess

yea, maybe a little hope is still present. Would be great to hear a feedback from the devs

**Comment by Bungeetaco** [comment]
Any updates on whether support for this is possible or not?

**Comment by verna123456** [comment]
is any progress for Ampere architectures?

**Comment by arisu3** [comment]
Yes, there is progress (machine translate may be required): https://blog.kkk.rs/archives/38

The 30 series and newer do not support the current vgpu_unlock techniques because they switched to a newer, more efficient virtualization method called SR-IOV that cannot be unlocked because of Nvidia's locked down vBIOS. That blog post describes a technique that they will release shortly which allows reverting 30 and newer series to the older Mdev virtualization method.

**Comment by Onepamopa** [comment]
I was hoping they found a way to patch the BIOS... SR-IOV is a lot better than the "old way"

**Comment by kristibektashi** [comment]
Any news for this? I have a 4060 and I assume the method to unlock should be the same as the 4090
